.circleci/config.yml

```diff
@@ -177,9 +178,9 @@ workflows:
       - docs
       - lint
       - unit-test
-      - deploy:
-          type: approval
-          requires:
-            - lint
-            - unit-test
-            - docs
+      # - deploy:
```

Comment:
As we're not using this atm, and is causing a non-green status, I have disabled this step.

We will get back on this for doing automated releases at some point.